plugins/katello/developers.md

```diff
@@ -26,6 +26,70 @@ The Katello bug triage process is designed to address Redmine issues that have b
   * [Untriaged Ready for Testing](http://projects.theforeman.org/projects/katello/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=release_id&op%5Brelease_id%5D=%21*&f%5B%5D=subproject_id&op%5Bsubproject_id%5D=%21*&f%5B%5D=status_id&op%5Bstatus_id%5D=%3D&v%5Bstatus_id%5D%5B%5D=7&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)
 
+### Triage Runner
+
+We have a few standing developers that switch off running triage. Typically it is myself or Justin but we try to get the current release owner to jump in and run it as well as encouraging others to take it on. Takes one or two to get the hang of it, but it is a good experience.
+
+### Frequency
+
+Triage is done once a week for an hour. As long as a week is not skipped, this has been enough time to get through every issue and leave 5-15 minutes extra. When a release is pending, or a new release has been sent out the volume does spike.
+
+### Issue States
+
+There are a few important states that issues can be in that are worth knowing about before going into the process:
+
+  * New: Unassigned issue
+  * Assigned: Developer has taken responsibility
+  * Needs more information: User who filed the issue has been asked for further information to help triage or debug the issue
+  * Needs design: The issue requires investigation by a developer
+  * Ready for Testing: An issue has an open PR
+  * Closed: An issue has been fixed
+
+### Release States
+
+There are a few important states the issue can be in with respect to the 'Release' field within Redmine. These release states have an impact not only a release itself but the triage process as a whole. A quick definition of each that will make more sense in the process section:
+
+  * Empty: No release has been set, this is the "untriaged" bucket
+  * Katello Backlog: Issues that have been put on the backlog and not scheduled for a release due to any number of reasons
+  * Katello Recycle Bin: Issues that have been rejected, or are a duplicate, or have been in 'Needs more information' for longer than a week
+  * Katello X.Y.Z: Standard releases for issues to be targeted at
+
+### Process
+
+The triage runner starts the meeting by going through sets of issues broken down into different groups by priority of the grouping. This is so that if time were to run out, the most critical categories of issues are triaged. This list (with easy links for the runner) are at [1]. The runner is in charge of ensuring the flow of the meeting, and updating issues or ensuring that someone on the call takes responsibility for updating an issue. This latter part sometimes occurs if a duplicate needs to be looked up, or another developer on the call can explain/ask the user for more information more clearly than the runner.
```

Comment:
Could we include the triage issue lists link inline?

Reply:
Only reason I didn't was so that the person running the meeting can quickly get to the links.

Reply:
I mean, what is [1]?

Reply:
Ohhh, yea, let me remove that since this was copied from an email.